### DISCUSSION TITLE
import ".py" file in python using Pyodide?

### DISCUSSION THREAD

**Discussion by RainMa22** [pyodide/pyodide#1808] [Q&A] [answered]
After reading the docs, there is not really a documentation about importing “.py” file in python.

For Example:
I have a file called "Utils.py";
can I use `import Utils` to import the file? If yes, where should I put the file?

any assistance will be greatly appreciated

**Comment by RainMa22** [comment, marked as answer]
Nevermind. I realized that I can create a wheel file using `setuptools` and I can load it using `micropip`.

**Reply by grimmerk** [reply]
or fetch it https://pyodide.org/en/stable/usage/faq.html#how-can-i-load-external-python-files-in-pyodide

**Reply by phorward** [reply]
Check out #1917 for this as well.
